//package com.sincetimes.redwar.game.comnunicate.protocol;


message FriendInfoLite 
{
    required string baseid 						= 1;  //uid
    required string first_name 					= 2;  //用户昵称
	optional string pic							= 3;  //头像
    required uint32 lvl 						= 4;  //等级
	optional bool   is_yellow_vip     			= 5 [default = false];  //是否是黄钻
    optional bool   is_yellow_year_vip 			= 6 [default = false];  //是否是年费黄钻
    optional int32  yellow_vip_level        	= 7 [default = 0];     //黄钻等级
    optional string 	totalpercharge 			= 8 ;		// 充值记录
 	optional int32 	lastLogonTime 				= 9 [default = 0];	 	//最后一次登录时间   
 		
	//Q+会员
	optional int32  QQDmdLvl     				= 10 [default = 0];     //QQ会员等级等级
	optional bool   isQQDmd      				= 11 [default = false]; //是不是QQ会员
	optional bool   isQQDmdYear  				= 12 [default = false]; //是不是年费QQ会员		
 	optional bool   sameRegion     				= 13; //是否是同区好友
 		
    optional string platid 						= 14;  //openid
	optional int32 finishCaptureTm 				= 15; // 结束被占领时间
	optional bool   is_sumptuous_yellow_vip 	= 16 [default = false];  //是否是豪华黄钻
    optional int32 viplevel    					= 17 [default = 0];       //所在分区的vip等级 
    optional int32 viptype    					= 18 [default = 0];       //所在分区的vip类型	
    
    //QQ大厅
    optional int32 QHallDmdLvl					= 19[default=0];					//蓝钻等级
    optional bool  isQHallDmd					= 20[default=false];			//是否蓝钻
    optional bool	 isQHallDmdYear 			= 21[default=false];			//蓝钻年费
    optional bool	 isHighQHallDmd 			= 22[default=false];			//豪华蓝钻
    optional bool	 isHighQHallDmeYear 		= 23[default=false];	//豪华年费
}
